pkg/cmd/codespace/create.go

```diff
@@ -296,3 +372,9 @@ func buildDisplayName(displayName string, prebuildAvailability string) string {
 
 	return fmt.Sprintf("%s%s", displayName, prebuildText)
 }
+
+func waitForEnter(r io.Reader) error {
```

Comment:
Unused?

Reply:
Ah yes thank you, its no longer used. Will remove